Atomic/storage.py

```diff
@@ -38,6 +41,29 @@ def reset(self):
         except:
             selinux.restorecon(root)
 
+    def modify(self):
+        try:
+            shutil.copyfile(self.dss_conf, self.dss_conf_bak)
+            if len(self.args.devices) > 0:
+                self._add_device(self.args.devices)
+            if self.args.driver:
+                self._driver(self.args.driver)
+            if util.call(["docker-storage-setup"]) != 0:
+                os.rename(self.dss_conf_bak, self.dss_conf)
+                util.call(["docker-storage-setup"])
+                raise ValueError("Not all devices could be added")
```

Comment:
The error message only makes sense for `--add-device`.  We might not need a message at all since dpcker-storage-setup should have explained itself already.